src/syntax/gats/judgments.jl

```diff
@@ -129,3 +131,62 @@ A description of the argument sorts for a term constructor, used to disambiguate
 multiple term constructors of the same name.
 """
 const AlgSorts = Vector{AlgSort}
+
+"""
+`AlgStruct`
+
+A declaration which is sugar for an AlgTypeConstructor, an AlgTermConstructor 
+which constructs an element of that type, and projection term constructors. E.g.
+
+    struct Cospan(dom, codom) ⊣ [dom:Ob, codom::Ob]
+      apex::Ob
+      i1::dom->apex
+      i2::codom->apex
+    end
+
+Is sugar for:
+
+    Cospan(dom, codom, apex, i1, i2)::TYPE 
```

Comment:
I think it should just be `Cospan(dom, codom)::TYPE`. `apex, i1, i2` are not part of the type; the whole point is that you could have multiple `Cospan(dom, codom)` with different fields!